yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -631,3 +632,39 @@ var uniqueNullsNotDistinctIssue = issue.Issue{
 func NewUniqueNullsNotDistinctIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
 	return newQueryIssue(uniqueNullsNotDistinctIssue, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
+
+var databaseOptionsPG15Issue = issue.Issue{
+	Type:        DATABASE_OPTIONS,
+	Name:        "Database options",
+	Impact:      constants.IMPACT_LEVEL_2,
+	Description: "Database options (%s) introduced in PostgreSQL 12 and later are not supported yet in YugabyteDB.",
+	Suggestion:  "",
+	GH:          "https://github.com/yugabyte/yugabyte-db/issues/25575",
+	DocsLink:    "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
+	// MinimumVersionsFixedIn: map[string]*ybversion.YBVersion{
+	// 	ybversion.SERIES_2_25: ybversion.V2_25_0_0,
+	// },// Not marking it as supported for 2.25 as it seems these might be actually not supported at least the locale and collation ones https://github.com/yugabyte/yugabyte-db/issues/25541
+}
+
+func NewDatabaseOptionsPG15Issue(objectType string, objectName string, sqlStatement string, options []string) QueryIssue {
+	issue := databaseOptionsPG15Issue
+	issue.Description = fmt.Sprintf(issue.Description, strings.Join(options, ", "))
+	return newQueryIssue(issue, objectType, objectName, sqlStatement, map[string]interface{}{})
+}
+
+var databaseOptionsPG17Issue = issue.Issue{
+	Type:        DATABASE_OPTIONS,
```

Comment:
let's have a different type for both these issues? Type should ideally be unique. Perhaps, name can be the same.